rules/linux/defense_evasion_binary_copied_to_suspicious_directory.toml

```
@@ -0,0 +1,73 @@
+[metadata]
+creation_date = "2023/08/29"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/08/29"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule monitors for the copying or moving of a system binary to a suspicious directory. Adversaries may copy/move 
+and rename system binaries to evade detection. Copying a system binary to a different location should not occur often,
+so if it does, the activity should be investigated. 
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "System Binary Copied and/or Moved to Suspicious Directory"
+risk_score = 47
+rule_id = "fda1d332-5e08-4f27-8a9b-8c802e3292a6"
+severity = "medium"
```

```suggestion
risk_score = 21
rule_id = "fda1d332-5e08-4f27-8a9b-8c802e3292a6"
severity = "low"
```
Optional, we can bump it back after getting some telemetry